.github/workflows/build-and-publish-adhoc.yml

```diff
@@ -18,6 +18,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      -
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.16.1'
```

Comment:
Is there a reason not to use Node 16? E.g., some developer's machine where it's hard to get Node 16 but easy to get 14?

Note that if we do move to Node 16.14.0 specifically (which is the latest Node 16.x at the moment), https://nodejs.org/en/blog/release/v16.14.0/#notable-changes lists something we'll need to be aware of if we import .json files (which right now we only do in Playwright tests AFAIK). Typescript 4.5 or later can handle that syntax: https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#import-assertions — and since we're using current versions of Typescript in Playwright tests, we should be fine there. So I don't see a drawback to moving to the latest Node version, rather than 14.

P.S. Same comment applies to the other places where we specify the Node version.

Reply:
Good point.  I have another branch where I bump Node to 16.14.0 (latest LTS) and that PR will follow as soon as this PR is merged.  Glad to hear about the improvements.